packages/core/src/Dialog/Dialog.styled.tsx

```diff
@@ -133,6 +135,10 @@ export const DialogOverlay = ({ scrimColor, sheet, children, zIndex }: Partial<D
   )
 }
 
+const SmoothTransitionBox = styled(Box)`
+  transition: all 0.3s ease-in-out;
```

Comment:
@wesleymartin85 is there a specific duration you want to use for this transition?